backend/resources/verify_email.py

```diff
@@ -0,0 +1,17 @@
+from flask_restful import Resource
+
+from model.users import UsersModel
+from model.verify_email import VerifyModel
+from utils.lock import lock
+
+
+class VerifyEmail(Resource):
+
+    def get(self, key):
```

Comment:
Milllor `POST `o `PUT`, ja que actulizes algo no observes
```suggestion
    def post(self, key):
```